pkg/network/config.go

```diff
@@ -14,6 +14,9 @@ const (
 	// CNIDeviceName is the default network device name and in
 	// reality should have an int appended to it (cni-podman4)
 	CNIDeviceName = "cni-podman"
+	// DefaultPodmanDomainName is used for the dnsname plugin to define
+	// a localized domain name for a created network
+	DefaultPodmanDomainName = "podman.io"
```

Comment:
This doesn't seem right - podman.io is a real domain with real hosts on it. If we're setting DNS search domain to this, we could get spurious resolutions.

We should do something that we're definitely not hosting anything on - `internal.podman.io` or similar

Reply:
say what you want. it is just a placeholder and figured we debate it now.

Reply:
I think a subdomain of `podman.io` is fine, let's just make sure it doesn't have anything hosted on it (and we don't start using it for hosting)

Reply:
Well we already do host content on podman.io, the main launching site for the tool.  Would it be better to do io.podman, and then hope no one registers that name...

Reply:
On further investigation, Docker seems to be doing some DNS magic to resolve without requiring anything more than a hostname. They add `options ndots:0` to container resolv.conf, and I believe that this means they don't append anything to hostnames in DNS.

That would probably require DNS plugin changes, so we might be too late here. Fixing this to something in an invalid TLD - `dns.podman` maybe - seems like a good option